test-data/src/test/resources/org/openmrs/module/fhir2/providers/MedicationWebTest_update.xml

```diff
@@ -10,11 +10,11 @@
 -->
 <Medication xmlns="http://hl7.org/fhir">
     <id value="1085AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" />
-    <extension url="https://fhir.openmrs.org/ext/medicine">
-        <extension url="https://fhir.openmrs.org/ext/medicine#maximumDailyDose">
+    <extension url="http://fhir.openmrs.org/ext/medicine">
+        <extension url="http://fhir.openmrs.org/ext/medicine#maximumDailyDose">
```

Comment:
why `http` in favor of `https`?

Reply:
Basically, convention (all FHIR URLs I've seen use `http` rather than `https`). Likewise, infrastructure-wise, its easy to "upgrade" from `http` -> `https`, but basically impossible to work the other way around.